.rubocop.yml

```diff
@@ -157,6 +160,7 @@ Lint/UnusedMethodArgument:
   Description: 'Checks for unused method arguments.'
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars'
   Enabled: true
+  AutoCorrect: true
```

Comment:
I had to add this to get these to get --auto-correct to work on these rules. I think it has something to do with the usage of `DisabledByDefault` but I'm not 100% sure